packages/common-assets/styles/scss/main.scss

```diff
@@ -20,6 +20,10 @@ $text-color: #111 !default;
 // Portal end
 @import "portal.scss";
 
+.ant-picker-calendar, .ant-picker-calendar .ant-picker-panel {
+  background-color: var(--vscode-editor-background) !important,
+}
```

Comment:
There is not really a nice way to adjust antd's calendar styles dynamically. So we need to circle around this and apply it this way.